packages/legend-graph/src/graph-manager/action/execution/ExecutionResult.ts

```diff
@@ -144,13 +144,17 @@ export const getTDSRowRankByColumnInAsc = (
   b: TDSRow,
   colIndex: number,
 ): number => {
-  const a1 =
-    a.values[colIndex] === null || a.values[colIndex] === undefined
-      ? -Infinity
-      : a.values[colIndex];
-  const b1 =
-    b.values[colIndex] === null || b.values[colIndex] === undefined
-      ? -Infinity
-      : b.values[colIndex];
-  return Number(guaranteeNonNullable(a1)) - Number(guaranteeNonNullable(b1));
+  const a1 = a.values[colIndex];
+  const b1 = b.values[colIndex];
+  if (a1 === null || a1 === undefined) {
+    return -1;
+  }
+  if (b1 === null || b1 === undefined) {
+    return 1;
+  }
+  if (typeof a1 === 'string' && typeof b1 === 'string') {
```

Comment:
cant we just do .toString() then just localCompare on all of them.

Reply:
tested this -- not working for number

Reply:
"10" < "2"

Reply:
please see https://stackoverflow.com/questions/10198257/comparing-2-strings-alphabetically-for-sorting-purposes
`"a5b".localeCompare("a21b", undefined, { numeric: true })`
we do this when comparing versions in studio. 
We also have `isString` functions in shared.